examples/trials/network-morphism/retrain/search_space.json

```diff
@@ -0,0 +1,14 @@
+{
+    "optimizer": {
+        "_type": "choice",
```

Comment:
I am not sure we need a train example here..

Reply:
Fine. I will consider it later.

Reply:
Remove it now.